sdk/python/kfp/components/_python_op.py

```diff
@@ -325,8 +333,8 @@ def annotation_to_type_struct(annotation):
         io_name = parameter.name
         if isinstance(
             parameter_annotation,
-            (InputArtifact, InputPath, InputTextFile, InputBinaryFile,
-             OutputArtifact, OutputPath, OutputTextFile, OutputBinaryFile)):
+            (io_types.InputArtifact, InputArtifact, InputPath, InputTextFile, InputBinaryFile,
```

Comment:
Do we still need the other `InputArtifact` next to the new one?

Reply:
Just temporarily. I will send a follow up clean-up PR. I didn't want to break the existing container entrypoint code. I'll migrate that one to use the new classes instead.

Reply:
sounds good. thanks!